src/openlcb/MemoryConfigClient.cxxtest

```diff
@@ -117,6 +117,40 @@ TEST_F(MemoryConfigClientTest, readsmallpart)
                      b->data()->payload.size()));
 }
 
+TEST_F(MemoryConfigClientTest, writelarge)
```

Comment:
* Where are the magic numbers (56, 75, 34, and 0x51) coming from? Perhaps define named constants that describe what they mean
* Please add a comment to these test describing the required behavior they're validating. It's not at all easy to figure that out from the test itself.

Reply:
Those numbers are arbitrary, chosen for the test, and the consistency ofthose numbers are only within this test function. e.g. we register our space at segment 0x51, then we are sending write requests to segment 0x51 from the client.
I write at address 34 for a length of 75 bytes, then i verify that data in the backing space at offset 34 showed up with the bytes I wrote.